trade-finance-manager-ui/templates/case/_macros/case-summary-ndb.njk

```diff
@@ -0,0 +1,116 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% import './ukef-deal-stage.njk' as ukefDealStage %}
+{% import './supplier-type.njk' as supplierType %}
+
+{% macro render(params) %}
+  {% set deal = params.deal %}
+  {% set tfm = params.tfm %}
+  {% set user = params.user %}
+
+  <div data-cy="case-summary-ndb" class="case-summary">
+    <div>
+      <div class="case-summary-deal-id">
+        <div class="label">Deal ID</div>
+        <div class="ukef-heading-l" data-cy="ukef-deal-id">{{ deal.details.ukefDealId | dashIfEmpty }}</div>
+      </div>
+
+      {% set supplierColumnClassName = 'case-summary-supplier' %}
+
+      {% if deal.submissionDetails.buyerName %}
+        {% set supplierColumnClassName = 'case-summary-supplier chevron-right' %}
+      {% endif %}
+
+      <div class="{{ supplierColumnClassName }}" data-cy="supplier-column">
+        <div class="label">
+          {{ supplierType.render(deal.submissionDetails.supplierType) }}
+        </div>
+        <div class="ukef-heading-l" data-cy="exporter-name">{{ deal.exporter.companyName | dashIfEmpty }}</div>
+      </div>
+
+      {% if deal.submissionDetails.buyerName %}
+        <div class="case-summary-buyer" data-cy="buyer-name-column">
+          <div class="label">Buyer</div>
+          <div class="ukef-heading-l" data-cy="buyer-name">{{ deal.submissionDetails.buyerName }}</div>
+        </div>
+      {% endif %}
+    </div>
+    <div class="case-summary-overview-container">
+      <div class="ukef-grid-column-one-sixth case-summary-overview-column">
+        <div class="case-summary-overview-item">
+          <div class="ukef-heading-xs">Product</div>
+          <div class="ukef-body-s" data-cy="ukef-product">{{ tfm.product | dashIfEmpty }}</div>
+        </div>
+
+        <div class="case-summary-overview-item">
+          <div class="ukef-heading-xs">Bank</div>
+          <div class="ukef-body-s" data-cy="bank-name">{{ deal.bank.name | dashIfEmpty }}</div>
+        </div>
+
+        <div class="case-summary-overview-item">
+          <div class="ukef-heading-xs">Date received</div>
+            <div class="ukef-body-s" data-cy="date-received">
+              {% if tfm.dateReceived %}
```

Comment:
Can we use `dashIfEmpty` plugin here please?

Reply:
done